README.md

```diff
@@ -54,7 +54,29 @@ export PORT=3000
 npm run build && npm run start
 ```
 
-### Tests
+## Tooling Dev Experience
+
+### Commitlint
+
+Nous utilisons [commitlint](https://commitlint.js.org/#/guides-local-setup) pour vérifier que les messages de commit suive bien la convention ce dernier nécessite l'installation de husky.
+
```

Comment:
Do we have to mention `npx husky install` as it is already in `devDependencies` ?